src/js/components/recipient/RecipientOverview.jsx

```diff
@@ -15,136 +17,147 @@ const propTypes = {
+        ? (
+            <button
+                className="recipient-overview__alternate-names-button"
+                onClick={props.showAlternateNamesRecipientModal}>
+                {`Also known by ${numberOfAlternateNames} other ${pluralizeAltNamesLabel}`} <FontAwesomeIcon icon={faCaretRight} />
+            </button>
+        )
+        : null;
 
         // Format the location data
-        let address = (
-            <td>Address not provided in source system</td>
+    let address = (
+        <td>Address not provided in source system</td>
+    );
+    if (recipient.location.streetAddress || recipient.location.regionalAddress || recipient.location.fullCongressionalDistrict) {
+        address = (
+            <td>
+                <div>{recipient.location.streetAddress}</div>
+                <div>{recipient.location.regionalAddress}</div>
+                <div>{recipient.location.fullCongressionalDistrict}</div>
+            </td>
         );
-        if (recipient.location.streetAddress || recipient.location.regionalAddress || recipient.location.fullCongressionalDistrict) {
-            address = (
-                <td>
-                    <div>{recipient.location.streetAddress}</div>
-                    <div>{recipient.location.regionalAddress}</div>
-                    <div>{recipient.location.fullCongressionalDistrict}</div>
-                </td>
-            );
-        }
+    }
 
-        // Format business types
-        let businessTypes = (
+    // Format business types
+    let businessTypes = (
+        <td>
+            Not provided in source system
+        </td>
+    );
+    if (recipient.businessTypes.length > 0) {
+        businessTypes = (
             <td>
-                Not provided in source system
+                {recipient.businessTypes.map((type, i) =>
+                    <div key={i}>{type}</div>)}
             </td>
         );
-        if (recipient.businessTypes.length > 0) {
-            businessTypes = (
-                <td>
-                    {recipient.businessTypes.map((type, i) =>
-                        <div key={i}>{type}</div>)}
-                </td>
-            );
-        }
+    }
 
-        return (
-            <div
-                id="recipient-overview"
-                className="recipient-section recipient-overview">
-                <h2 className="recipient-overview__title">
-                    {recipient.name}
-                    {viewAlternateNames}
-                </h2>
-                <hr className="results-divider" />
-                <div className="recipient-overview__content">
-                    {parent}
-                    <div className="recipient-overview__children">
-                        {label}
-                        {viewChildren}
-                    </div>
-                    <div className="recipient-section__row">
+    const removeAllNonNumericCharacters = (str) => str.replace(/\D/g, '');
```

Comment:
This can go outside the definition of the component, seems like it should be a helper or something. I think lodash maybe does this too?

Reply:
I removed it because it's not needed anymore. In the AC, Ross mentioned that it's fine to always show the results even if the # of transactions are 0.